blueprints/automation/panhans/heating_control.yaml

```diff
@@ -335,12 +352,15 @@ variables:
     "{{ (as_timestamp(state_attr(active_scheduler,'next_event'),as_timestamp(now()))
     - as_timestamp(now())) / (60 * 60) <=  presence_time_window }}"
   temperatur:
-    "{% if ( states(active_scheduler) == 'on' and ( persons_home_count |
-    int > 0 or guest_state == true)) %}\n\n  {% if ( presence_detected == false and
-    is_presence_time_window == true ) %}\n    {{ min_temp }}\n  {% else %}\n    {{
-    states(set_temp) }}\n  {% endif %}\n\n{% else %}\n\n  {% if party_state == true
-    %}\n    {{ states(set_temp) }}\n  {% else %}\n    {{ min_temp }}\n  {% endif %}\n\n{%
-    endif %}\n"
+    "{% if party_state == true and party_temp != none %}\n  {{ states(party_temp) }}\n
```

Comment:
Put the party temperature inside the party if clause like:

``` Jinja
{% if ( states(active_scheduler) == 'on' and ( persons_home_count | int > 0 or guest_state == true)) %} 
  {% if ( presence_detected == false and is_presence_time_window == true ) %} 
    {{ min_temp }}
  {% else %}
    {{ states(set_temp) }}
  {% endif %}
{% else %} 
  {% if party_state == true %}
    {{ iif(party_temp != none, states(party_temp), states(set_temp)) }}
  {% else %}
    {{ min_temp }}
  {% endif %}
{% endif %}
```

Reply:
This was my first thought and then I realized, it wouldn't overwrite set_temp when heating would occur regularly. So I will do (kinda) this and move the party_state clause to the beginning.

Reply:
This works for me:

```
temperatur: >
    {% if ( states(active_scheduler) == 'on' and ( persons_home_count | int > 0 or guest_state == true)) %} 
      {% if ( presence_detected == false and is_presence_time_window == true ) %} 
        {{ min_temp }}
      {% else %}
        {{ states(set_temp) }}
      {% endif %}
    {% else %} 
      {% if party_state == true %}
        {% if party_temp != none %}
          {{ states(party_temp) }}
        {% else %}
          {{ states(set_temp) }}
        {% endif %}
      {% else %}
        {{ min_temp }}
      {% endif %}
    {% endif %}
```

But the _input_numer_ also must be added to the trigger section in case you want to change temperature.

Reply:
Ok, I understand. Now lets change it to:

``` yaml
temperatur: >
    {% if party_state == true %}
      {% if party_temp != none %}
        {{ states(party_temp) }}
      {% else %}
        {{ states(set_temp) }}
      {% endif %}
    {% else %}
      {% if ( states(active_scheduler) == 'on' and ( persons_home_count | int > 0 or guest_state == true)) %} 
        {% if ( presence_detected == false and is_presence_time_window == true ) %} 
          {{ min_temp }}
        {% else %}
          {{ states(set_temp) }}
        {% endif %}
      {% else %} 
        {{ min_temp }}
      {% endif %}
    {% endif %}
```

This should fit your requirements